languages/python/sqlalchemy-oso/tests/test_roles2.py

```diff
@@ -2209,6 +2212,65 @@ def test_user_in_role(
         assert repo.org_id == "osohq"
 
 
+def test_mismatched_id_types_throws_error(engine, Base, User):
+    class One(Base):
+        __tablename__ = "ones"
+
+        id = Column(String(), primary_key=True)
+
+    class Two(Base):
+        __tablename__ = "twos"
+
+        id = Column(Integer(), primary_key=True)
+
+    Session = sessionmaker(bind=engine)
+
+    oso = SQLAlchemyOso(Base)
+
+    with pytest.raises(OsoError):
+        oso.enable_roles(User, Session)
+
+
+def test_integer_ids(engine, Base, User):
```

Comment:
would be cool to parametrize this like

```suggestion
@pytest.mark.parametrize("sa_type,one_id", [(String, "1"), (Integer, 1)])
def test_integer_ids(engine, Base, User, sa_type, one_id):
```

and then update the body to use those two parametrized fixtures

Reply:
this is cool